optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -293,6 +337,56 @@ export const TrialTable: FC<{
             />
           </Box>
         </Grid>
+        {trials[index].state === "Running" ? (
+          <Grid item xs={12}>
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Trial tell operations
```

Comment:
Cosmetic suggestion, I believe this communicates well enough and is more compact. What do you think?
```suggestion
                Trial tell
```